### DISCUSSION TITLE
MAS wont open suddenly

### DISCUSSION THREAD

**Discussion by Wraxend25** [Monika-After-Story/MonikaModDev#9795] [QA - tech support + general]
I had opened MAS and then I looked over and it had closed with something possibly gone. Now it won't open. When I click ddlc.exe it says 

Could not execute C:/users/andre/onedrive/documents/monikaaftwrstory/ddlc-1.1.1-pc/lib/windows-i686/ddlc.exe. is it missing?

Please advise

**Comment by Wingdinggaster656** [comment]
> Could not execute C:/users/andre/onedrive/documents/monikaaftwrstory/ddlc-1.1.1-pc/lib/windows-i686/ddlc.exe. is it missing?

I noticed `onedrive`. You should move MAS to another place, immediately. Onedrive is known to have issues about this.

If after that, the problem still didn't get solved, you could try to have a 100% reinstallation. That would probably fix it.
